runtime/compiler/control/DLLMain.cpp

```diff
@@ -325,6 +325,7 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void * reserved)
                   }
 
                jitConfig = vm->jitConfig;
+               ::jitConfig = jitConfig;
```

Comment:
I'm trying to grok what this means.  Do you have the assignment reversed?  I don't think the local `jitConfig` (the RHS) has been initialized to anything other than NULL at this point.